### PR TITLE
bump(github.com/openshift/openshift-sdn) b74a40614df17a99b8f77ac0dbd9

### PR DESCRIPTION
**Pull request by pravisankar**
Depends on https://github.com/openshift/openshift-sdn/pull/313

### CODE REVIEW

**Comment by pravisankar** [comment]
@openshift/networking PTAL

File: pkg/cmd/server/api/v1/conversions.go [hunk 1]
```diff
@@ -88,7 +88,7 @@ func addDefaultingFuncs(scheme *runtime.Scheme) {
 				obj.NetworkConfig.MTU = 1450
 			}
 			if len(obj.IPTablesSyncPeriod) == 0 {
-				obj.IPTablesSyncPeriod = "5s"
```
Comment (smarterclayton) [inline pkg/cmd/server/api/v1/conversions.go]: What's this used for?  Is this a delay before endpoints are changed?

Comment (pravisankar) [inline pkg/cmd/server/api/v1/conversions.go]: No. On the node, k8s periodically syncs proxy iptables rules (to ensure it's not overwritten by other tools or by hand) and the frequency of this sync loop is driven by this IPTablesSyncPeriod config. Originally, k8s started with 5s and now they changed it to 30s.

Comment (smarterclayton) [inline pkg/cmd/server/api/v1/conversions.go]: Ok

**Comment by dcbw** [comment]
LGTM

**Comment by pravisankar** [comment]
[test]

**Comment by pravisankar** [comment]
[test]

**Comment by pravisankar** [comment]
[merge]

**Pull request merged**